src/Core/AspectKernel.php

```diff
@@ -263,6 +293,18 @@ protected function registerTransformers()
     }
 
     /**
+     * An opportunity to add/remove some transformers depending on project needs
+     *
+     * @param array|SourceTransformer[] $defaultTransformers
+     *
+     * @return array|\Go\Instrument\Transformer\SourceTransformer[]
+     */
+    protected function customizeTransformers (array $defaultTransformers)
```

Comment:
This method is not needed, if you want to customize the list of transformers, just override the `registerTransformers()` method. AspectMock has an example: https://github.com/Codeception/AspectMock/blob/master/src/AspectMock/Kernel.php#L62-L68